caom2/caom2/tests/test_obs_reader_writer.py

```diff
@@ -449,15 +476,165 @@ def compare_planes(self, expected, actual, version):
                                     actual_plane.provenance)
             self.compare_metrics(expected_plane.metrics, actual_plane.metrics)
             
-            self.compare_time(expected_plane.time, actual_plane.time)
-            
             if version == 21:
                 self.compare_quality(expected_plane.quality,
                                      actual_plane.quality)
+            
+            if version >= 22:
+                self.compare_position(expected_plane.position, actual_plane.position)
+                self.compare_energy(expected_plane.energy, actual_plane.energy)
+                print('comparing time')
+                self.compare_time(expected_plane.time, actual_plane.time)
+                print('compared time')
+                self.compare_polarization(expected_plane.polarization, actual_plane.polarization)
 
             self.compare_artifacts(expected_plane.artifacts,
                                    actual_plane.artifacts, version)
-
+            
+    def compare_none_or_equal(self, expected, actual, field):
+        if expected is None:
```

Comment:
Is this necessary? assertEquals does not return True for None, None combination?